package.json

```diff
@@ -20,9 +20,11 @@
         "@babel/register": "^7.12.13",
         "@tsconfig/node14": "^1.0.1",
         "@types/jasmine": "3.6.2",
+        "@wdio/appium-service": "^6.12.1",
         "@wdio/cli": "^6.10.10",
         "@wdio/jasmine-framework": "^6.11.0",
         "@wdio/local-runner": "^6.12.1",
+        "@wdio/selenium-standalone-service": "^6.12.1",
```

Comment:
question: why do we need this dependency for?

Reply:
it is the service needed local run